app/extensions/contact/slack.py

```diff
@@ -15,7 +16,7 @@ def load():
 class SLACK(Extension):
```

Comment:
```suggestion
class Slack(Extension):
```

I noticed you were doing some code quality changes, we could change this to normal capitalization, instead of all caps

Reply:
Sounds great - I went ahead and changed that